### ISSUE TITLE
Directory-based collections

### ISSUE THREAD

**Issue by shreyasminocha** [11ty/eleventy#487]
Can I configure a directory to function as a collection rather than having to set the `tags` key in the frontmatter of every file in the certain directory?

For example, all files in my `committees/` directory have the `tags` key set to `committees`. I find this unnecessarily repetitive.

**Comment by shreyasminocha** [comment]
Found `addCollection` in docs.

**Comment by kleinfreund** [comment]
I think what you want for this use case are data directories: https://www.11ty.io/docs/data-template-dir/

in the `committees` directory, create a `committees.json` file and add:

```json
{
  "tags": "committees"
}
```

Also, absolutely no need to apologize for asking a question. :)

**Comment by shreyasminocha** [comment]
@kleinfreund Interesting, thanks! That looks like what I was looking for. The terminology threw me off a bit 😅

**Comment by kleinfreund** [comment]
Right, now that I think about it, it’s one thing to imply everything that belongs to a tag gets to be part of a collection with the same name and a completely other thing to want a collection without thinking about tags.

**Comment by AlahmadiQ8** [comment]
Hi,

I found this issue while googling `directory based collections eleventy`.

unfortunately, when i try to use this method, I cannot access that tag top input directoty. 

here is an example I have the following structure

```
eleventy-src
├── _includes
│   └── layouts
│       └── base.njk
├── eleventy-src.11tydata.js
├── index.njk
└── posts
    ├── 2018-03-10-on-prioritizing-side-projects.md
    ├── 2018-05-21-fear-of-looking-stupid.md
    └── posts.json
```

posts.json consists of the following: 
```json 
{
  "tags": [ "posts" ]
}
```

in the `index.njk` file, `collections.posts` do not exists. 
I also tried debuging and printing the tags in a particular posts and the tag 'posts' is still not applied.

am i missing a step?
